scalableminds/webknossos
# Enforce Pricing for Task/Project Management Features

This PR adds restrictions to the availability of certain features depending on your organization's plan. In particular, it prevents unauthorized access to Task & Projects etc.

Changes:
- Added current orga to wk store, including Redux actions/reducers/...
- Enforce pricing in navbar links
- Enforce pricing in router
- Added a small new page when you are not authorized to access a feature
- Enforce "Add Folder" in dashboard

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Check that the correct features were restricted and with the right plan
- Change your orgas plan to Basic/Team/Power... in the DB and check if the restrictions apply

### Issues:
- contributes to #6509 

------
(Please delete unneeded items, merge only when none are left open)
